src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -144,6 +144,8 @@
             + "The value of '${fieldName}' field should be no longer than ${maxLength} characters.";
     public static final String INVALID_NAME_ERROR_MESSAGE =
             ERROR_INFO + " " + HINT_FOR_CORRECT_FORMAT_FOR_INVALID_NAME;
+    public static final String TEAM_NAME_IS_VALID_EMAIL_ERROR_MESSAGE =
```

Comment:
The change to this file should be exactly the same as https://github.com/TEAMMATES/teammates/commit/df87c34d9652d2c1f7fb77b10c45fc03b6656cb2?